logging/cluster-logging-release-notes.adoc

```diff
@@ -372,21 +386,25 @@ This release includes link:https://access.redhat.com/errata/RHSA-2021:5129[RHSA-
 
 [id="cluster-logging-release-notes-5-3-0_{context}"]
 == OpenShift Logging 5.3.0
-The following advisories are available for OpenShift Logging 5.3.x:
+This release includes link:https://access.redhat.com/errata/RHSA-2021:4627[RHSA-2021:4627 OpenShift Logging Bug Fix Release 5.3.0]
 
 [id="openshift-logging-5-3-0-new-features-and-enhancements_{context}"]
 === New features and enhancements
-* With this update, authorization requirements for Log Forwarding have been relaxed. Outputs may now be configured with SASL, username/password, or TLS.
+* With this update, authorization options for Log Forwarding have been expanded. Outputs may now be configured with SASL, username/password, or TLS.
```

Comment:
```suggestion
* With this update, authorization options for Log Forwarding have been expanded. Outputs can now be configured with SASL, username/password, or TLS.
```

Reply:
ISG: may verb
Use only to indicate permission. Otherwise, use "can" to indicate ability, or use "might" to indicate possibility.
"May" is overused in technical information. In most cases, use "can" or "might" instead.